Pod/Classes/MenuItemView.swift

```diff
@@ -162,6 +207,28 @@ public class MenuItemView: UIView {
         
         addSubview(dividerImageView)
     }
+    
+    private func layoutMultiLineLabel() {
+        let viewsDictionary = ["titleLabel": titleLabel, "descLabel": descLabel]
+        let metrics = ["margin": options.menuItemMargin,
+                       "titleHeight": options.menuTitleHeight,
+                       "descHeight": options.menuDescHeight
+                       ]
+        
+        let labelSize = calculateLabelSize()
+        
+        let horizontalConstraints = NSLayoutConstraint.constraintsWithVisualFormat("H:|[titleLabel]|", options: [], metrics: metrics, views: viewsDictionary)
+        let verticalConstraints = NSLayoutConstraint.constraintsWithVisualFormat("V:|-[titleLabel(titleHeight)][descLabel(descHeight)]-|", options: [], metrics: metrics, views: viewsDictionary)
+        
+        let descHorizontalConstraints = NSLayoutConstraint.constraintsWithVisualFormat("H:|[descLabel]|", options: [], metrics: nil, views: viewsDictionary)
+        
+        NSLayoutConstraint.activateConstraints(horizontalConstraints + verticalConstraints + descHorizontalConstraints)
+        
+        widthConstraint = NSLayoutConstraint(item: titleLabel, attribute: .Width, relatedBy: .Equal, toItem: nil, attribute: .Width, multiplier: 1.0, constant: labelSize.width)
+        widthConstraint.active = true
+        widthConstraint = NSLayoutConstraint(item: descLabel, attribute: .Width, relatedBy: .Equal, toItem: nil, attribute: .Width, multiplier: 1.0, constant: labelSize.width)
+        widthConstraint.active = true
```

Comment:
`widthConstraint` is for `titleLabel`. Two constraint properties for both `titleLabel` and `descriptionLabel` are needed to change width dynamically.